pages/docs/installation/installation-special-systems.md

```diff
@@ -794,3 +794,75 @@ export PRECICE_BOOST_ROOT=$HOME/software/boost_1_53_0
 * configure ALYA with `-L[PathToPreCICE]/build/last -lprecice -lstdc++ -lrt`
 * for running: also put `module load intel` in your jobscript
 * use `network="ib0"` for sockets communication beyond one node
+
+### Max Planck Computing and Data Facility (MPCDF) Cobra cluster
+
+#### Installing dependencies
+
+##### Eigen3
+
+The Eigen repository can be transferred to the cluster and extracted. The path of the extracted folder is set in `~/.bashrc`.
+
+##### Boost
+
+Boost installation needs to be done with the GCC compiler. By default the Intel compilers will be loaded in the session. After the Boost source is downloaded and transferred to the cluster, the following commands are run to install Boost:
```

Comment:
Please switch to a more active/casual language style, see https://precice.org/docs-meta-overview.html#language--style

What helps to find the right level it to think as is you are explaining the content to a close colleague/friend over coffee.

For example, I would rewrite this part as follows:

```suggestion
Transfer the Eigen sources to the cluster and set the path to your `~/.bashrc`. Read more details in the [installing dependencies](./installation-source-dependencies.html#eigen).

##### Boost

You need to [build Boost from source](https://precice.org/installation-source-dependencies.html#boost) with the GCC compiler. After uploading the Boost sources to the cluster, run the following:
```

Also: why does the user _need_ to build with GCC? Did you have issues with Intel? If a cluster prefers a compiler, we should try to stick to that, unless we document the situation.